README.md

```diff
@@ -97,4 +97,4 @@ Ron Pham - [@Bacoburn](https://github.com/Bacoburn)
 
 Adrian Knight - [@Ajknight121](https://github.com/Ajknight121)
 
-Ares Stavropoulos - [@Ares1605](https://github.com/Ares1605)
\ No newline at end of file
+Ares Stavropoulos - [@Ares1605](https://github.com/Ares1605)
```

Comment:
I am going to ignore this for now, but double check that your IDE isn't messing with things.

Reply:
I don't know why this is being changed, because I haven't modified anything from the ReadMe file